src/Card.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './card.css';
+
+function Card (props){ 
+    const [ email, setEmail ] = React.useState('Loading...');
+    const [ firstName, setFirstName ] = React.useState('Loading...');
+    const [ lastName, setLastName ] = React.useState('Loading...');
+    const [ avatar, setAvatar ] = React.useState('Loading...');
```

Comment:
Using the `'Loading...'` string here will actually cause the image to look broken when the page first loads, since it's not a valid src. One cool trick is to use a default image, like a picture of a loader or a blank avatar, as the default.